lib/ui/platform_dispatcher.dart

```diff
@@ -249,6 +248,28 @@ class PlatformDispatcher {
     _onMetricsChangedZone = Zone.current;
   }
 
+  FlutterView _createView(int id) {
+    return FlutterView._(id, this, const _ViewConfiguration());
+  }
+
+  void _addView(int id) {
+    assert(id != _implicitViewId, 'The implicit view #$id can not be added.');
+    _doAddView(id);
+    _invoke(onMetricsChanged, _onMetricsChangedZone);
+  }
+
+  void _doAddView(int id) {
```

Comment:
I think this would be a little easier to read without `_doAddView`:

```dart
  PlatformDispatcher._() {
    _setNeedsReportTimings = _nativeSetNeedsReportTimings;
    if (_implicitViewId != null) {
      _views[_implicitViewId] = FlutterView._(id, this, const _ViewConfiguration());
    }
  }

...

  void _addView(int id) {
    assert(id != _implicitViewId, 'The implicit view #$id can not be added.');
    assert(!_views.containsKey(id), 'View ID $id already exists.');
    FlutterView._(id, this, const _ViewConfiguration());
    _invoke(onMetricsChanged, _onMetricsChangedZone);
  }
```

I don't feel strongly about this, feel free to skip

Reply:
I think this method contains sufficient parts that I don't want to repeat it. I've renamed it to `_insertNewView` and moved it to before `_addView`. Hopefully it's a bit better.